src/analysis/scripts/export_connectivity.sh

```diff
@@ -98,7 +98,7 @@ function ec_export_destination_geojson() {
   ogr2ogr -f GeoJSON "${FILENAME}" \
           -t_srs EPSG:4326 \
           "PG:host=${NB_POSTGRESQL_HOST} dbname=${NB_POSTGRESQL_DB} user=${NB_POSTGRESQL_USER}" \
-          -sql "select *, geom_pt from ${EXPORT_TABLENAME}"
+          -sql "select *, geom_pt from ${EXPORT_TABLENAME} where not ST_IsEmpty(geom_pt)"
```

Comment:
Yup, `-skipfailures` is way better.  Covers more cases, plus it logs errors but still exits normally.  Pushed a commit to switch to that.  I'll rebase the first one out of existence before merging.